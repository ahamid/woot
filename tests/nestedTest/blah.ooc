main: func() {
    "End the rain will kill us all" println()
}

