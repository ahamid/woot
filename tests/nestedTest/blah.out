End the rain will kill us all
